### PR TITLE
[pipeline](exec) disable shared scan in default and disable shared scan in limit with where scan

### PR DESCRIPTION
**Pull request by HappenLee**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by HappenLee** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java [hunk 1]
```diff
@@ -721,6 +723,10 @@ public class SessionVariable implements Serializable, Writable {
 
     @VariableMgr.VarAttr(name = ENABLE_PIPELINE_X_ENGINE, fuzzy = false, varType = VariableAnnotation.EXPERIMENTAL)
     private boolean enablePipelineXEngine = false;
+
+    @VariableMgr.VarAttr(name = ENABLE_SHARED_SCAN, fuzzy = false, varType = VariableAnnotation.EXPERIMENTAL)
+    private boolean enableSharedScan = false;
```
Comment (yiguolei) [inline fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java:728]: should  set forward = true, for example insert into select is forward to master

**Comment by HappenLee** [comment]
run buildall

**Comment by HappenLee** [comment]
run buildall

**Review by Gabriel39 — APPROVED** [review]

**Review by zclllyybb — APPROVED** [review]

**Pull request merged**